README.md

```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+# Bookstore
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+This project is built with:
 
-### `npm start`
+- ![React](https://img.shields.io/badge/-React-000000?style=flat&logo=react)
+- ![JavaScript](https://img.shields.io/badge/-JavaScript-000000?style=flat&logo=javascript)
+- ![HTML](https://img.shields.io/badge/-HTML-000000?style=flat&logo=html)
+- ![Sass](https://img.shields.io/badge/-Sass-000000?style=flat&logo=sass&logoColor=ffffff&labelColor=%23CC6699)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+Click [here](https://tresorsawasawa.github.io/Bookstore/) to see the live demo
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+To get the content of this project locally you need to run this command in your terminal:
 
-### `npm run build`
+- `git clone hhttps://github.com/tresorsawasawa/Bookstore.git`
+- `cd math_magicians`
```

Comment:
- [OPTIONAL] Please this project is about `Bookstore`, kindly update the above command accordingly.

Reply:
Thank You for the Review